framework/.eslintrc.json

```diff
@@ -30,9 +30,6 @@
 				]
 			}
 		],
-		"callback-return": "error",
```

Comment:
You've removed more rules than the commit indicates. 

Besides, there are many Eslint disable statements. I would prefer to leave the rule until we're really able to remove them. Otherwise, we should create an issue to act on all this Eslint disables comments.

Reply:
If we add `off`, there will be more code that uses the invalid rules.
Therefore, I would prefer to turn it off, and fix the rule one by one when we do refactoring